internal/middleware/storeid.go

```diff
@@ -0,0 +1,77 @@
+package middleware
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+	"google.golang.org/grpc"
+)
+
+const (
+	storeIDCtxKey ctxKey = "store-id-context-key"
+
+	storeIDTraceKey = "store_id"
+)
+
+func StoreIDFromContext(ctx context.Context) (string, bool) {
```

Comment:
Mind adding a `storeid_test.go` file and covering these signatures with some simple tests?